frontend/src/app/settings/modelsources/t4c-settings/edit-t4c-instance/edit-t4c-instance.component.ts

```diff
@@ -151,6 +154,22 @@ export class EditT4CInstanceComponent implements OnInit, OnDestroy {
     }
   }
 
+  toggleArchive(): void {
+    if (this.instanceId) {
+      this.t4cInstanceService
+        .updateInstance(this.instanceId, {
+          is_archived: !this.isArchived,
+        })
+        .subscribe((instance) => {
+          this.isArchived = instance.is_archived;
+          this.toastService.showSuccess(
+            'Instance updated',
+            `The instance “${instance.name}” is now archived.`
```

Comment:
possible that I missed something but shouldn't it be something like
```suggestion
            'Instance updated',
            `The instance “${instance.name}” is now ${this.isArchived? "archived" : "unarchived"}.`
```

Reply:
Good catch!